### DISCUSSION TITLE
Hooks commit merge

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#232] [Team Posts]
@reactjs/reactjs-org-localization Hi everyone!

I ran the update bot on the recent commit for the hooks release, so you should be getting a bunch of [Merge] issues from the react translation bot. Unfortunately it doesn't seem like any of the issues were able to be merged, so I'll take a look at that tomorrow

This is our big first test to see how this approach will work :)

**Comment by tesseralis** [comment]
I ended up merging things manually and you should be able to find a pull request called "Sync Original" that does the update.

**Comment by alejandronanez** [comment]
Oh boy, that PR is huge 😟

**Comment by alejandronanez** [comment]
@dmoralesm and @carburo, does holding off any merge until we merge https://github.com/reactjs/es.reactjs.org/pull/115 sound like a good plan to you?

I think that's the only way to avoid more conflicts.

**Comment by tesseralis** [comment]
What do people think about this behavior for the bot? Instead of cherry picking each commit into a new PR, just run it every day or so and create a PR with all the changes. Or possibly a different PR per file?

**Comment by smikitky** [comment]
Is the following possible?

- If commits are from one PR, create one PR that contains the same set of commits
- If someone directly pushed a commit, create a PR with that one commit

**Comment by tesseralis** [comment]
I think that's something we have to do in the original repo -- require squash commits and prevent merge/rebase commits. I could put up an issue to see if thats's feasible.
